File: MyBreak/DayThree/DayFour/JavaScript/FindTheTwoLargestNumbers.js


function FindTheTWOLargestNumbers(){
  
  let temp = 0
  let arr = []
  let size = arr.length; 

  for (let i = 0; i <  size; i++){

    let value = prompt("Enter a number between  ")
    for(let j = 0; j < size+1; j++){

      if(arr[j] > arr[i]){
        temp = arr[j]
        arr[j] = arr[i]
        arr[i] = temp

        
      }
    }
  }
}

File: MyBreak/DayThree/DayFour/Java/FindTheTwoLargestNumbers.java
import java.util.*;

public class FindTheTwoLargestNumbers {

  public static void main(String[] args) {
    FindTheTwoLargestNumbers();
  }

  public static void FindTheTwoLargestNumbers() {

    Scanner scan = new Scanner(System.in);
    int[] numbers = new int[10];
    int temp = 0;
    int size = numbers.length;

    for (int index = 0; index < size; index++) {
      System.out.println("Enter  a number: ");
      numbers[index] = scan.nextInt();

      for (int indexTwo = 0; indexTwo < size - 1; indexTwo++) {
        if (numbers[indexTwo] > numbers[indexTwo + 1]) {
          temp = numbers[indexTwo];
          numbers[indexTwo] = numbers[indexTwo + 1];
          numbers[indexTwo + 1] = temp;
        }
      }
      for (int number : numbers) {
        System.out.println(number);
      }
    }

  }
}
